components/PicklistTable/index.tsx

```diff
@@ -102,21 +114,29 @@ export default function PickListTable({
 							<TableHead className='text-center font-semibold' colSpan={2}>Endgame</TableHead>
 						</TableRow>
 						<TableRow>
-							<TableHead>Number</TableHead>
-							<TableHead>First Pickability</TableHead>
-							<TableHead>Second Pickability</TableHead>
-							<TableHead>Amp</TableHead>
-							<TableHead>Speaker</TableHead>
-							<TableHead>Amp</TableHead>
-							<TableHead>Speaker</TableHead>
-							<TableHead>Amplified Speaker</TableHead>
-							<TableHead>Climb Rate</TableHead>
-							<TableHead>Trap Notes</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary'>Number</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('firstPickability')}>First Pickability</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('secondPickability')}>Second Pickability</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('autoAmpNotes')}>Amp</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('autoSpeakerNotes')}>Speaker</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('teleopAmpNotes')}>Amp</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('teleopSpeakerNotes')}>Speaker</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('teleopAmplifiedSpeakerNotes')}>Amplified Speaker</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('climbRate')}>Climb Rate</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('trapNotes')}>Trap Notes</TableHead>
```

Comment:
Let's create a mapping function that will do this, for example, see the NavBar component.

Essentially what it'll look like is we have a constant list of objects like:
```javascript
const headers = [
 { id: "firstPickability", title: "First Pickability" },
 { id: "secondPickability", title: "Second Pickability" },
...
]
```

And then we can just replace all the TableHeads with:
```javascript
headers.map(({id, title}) => <TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick(id)}>{title}</TableHead>)
```

This just makes any future edits of the style/each TableHead a bit easier

Reply:
Only other thing is make sure each TableHead has a key prop as well

Reply:
Got this one done